targets/drive/web/services.jsx

```diff
@@ -40,9 +40,16 @@ document.addEventListener('DOMContentLoaded', () => {
   const { intent } = getQueryParameter()
 
   const client = new CozyClient({
-    cozyURL: cozyUrl,
+    uri: cozyUrl,
```

Comment:
Do we want to change this now? I think after this PR we will still be using the old CozyClient, so it should be called `cozyURL`, no?

Reply:
Hmmm... with this PR, cozy-client is not used anymore by the intents, only cozy-client-js is used. So I should get rid of this client instantiation right?

Reply:
Oh right we're in the services `main.jsx`! Yeah ok then I guess we don't need it at all.